website/sidebars.json

```diff
@@ -50,5 +50,14 @@
       "sharing-dark",
       "changelog"
     ]
+  },
+  "Slack": {
```

Comment:
Based on Google analytics, some set of people have found it, but I agree it's not ideal. Do you think shifting it to the left would help? I think that's easy to do.

I definitely think it needs its own space because of the amount of content/navigation/etc we want to put in there... I might be able to make a sidebar link open it up as well, but on my screen the sidebar is cut off after Language -> Packages.

Reply:
Maybe keep it on the topbar and also add a link or section on the left?